go.mod

```diff
@@ -8,14 +8,20 @@ require (
 	github.com/c2h5oh/datasize v0.0.0-20171227191756-4eba002a5eae
 	github.com/dgryski/go-gk v0.0.0-20140819190930-201884a44051
 	github.com/dgryski/go-lttb v0.0.0-20180810165845-318fcdf10a77
+	github.com/gonum/blas v0.0.0-20181208220705-f22b278b28ac // indirect
+	github.com/gonum/diff v0.0.0-20181124234638-500114f11e71 // indirect
+	github.com/gonum/floats v0.0.0-20181209220543-c233463c7e82 // indirect
+	github.com/gonum/integrate v0.0.0-20181209220457-a422b5c0fdf2 // indirect
+	github.com/gonum/internal v0.0.0-20181124074243-f884aa714029 // indirect
+	github.com/gonum/lapack v0.0.0-20181123203213-e4cdc5a0bff9 // indirect
+	github.com/gonum/mathext v0.0.0-20181121095525-8a4bf007ea55 // indirect
+	github.com/gonum/matrix v0.0.0-20181209220409-c518dec07be9 // indirect
+	github.com/gonum/stat v0.0.0-20181125101827-41a0da705a5b // indirect
```

Comment:
Can you explain these changes?

Reply:
These are coming with `go mod tidy`, basically it is trying to cleanup the dependencies declaration in the `go modules` fashion.